seqr/utils/search/hail_search_utils.py

```diff
@@ -186,3 +186,18 @@ def _validate_expected_families(results, expected_families):
             f'{variant_id} ({"; ".join(sorted(families))})' for variant_id, families in invalid_family_variants
         ])
         raise InvalidSearchException(f'Unable to return all families for the following variants: {missing}')
+
+
+MAX_FAMILY_COUNTS = {Sample.SAMPLE_TYPE_WES: 200, Sample.SAMPLE_TYPE_WGS: 35}
+
+
+def validate_hail_backend_no_location_search(samples):
+    sample_counts = samples.filter(dataset_type=Sample.DATASET_TYPE_VARIANT_CALLS).values('sample_type').annotate(
+        family_count=Count('individual__family_id', distinct=True),
+        project_count=Count('individual__family__project_id', distinct=True),
+    )
+    from seqr.utils.search.utils import InvalidSearchException
```

Comment:
Should this import be at the top of the file now that we use the exception twice?

Reply:
no I tried and we end up in circular dependency hell, and I thought refactoring where it lived was out of scope of this PR

Reply:
Ah.  Alright.  We should make a ticket though!